### ISSUE TITLE
Wrong type definition for coordinatesContainPoint

### ISSUE THREAD

**Issue by sds-dubois** [Esri/terraformer#342]
[Here](https://github.com/Esri/terraformer/blob/master/terraformer.d.ts#L279) it says that `coordinatesContainPoint` takes an array of `Coordinates`, ie `Coordinate[][]`:

```
public static coordinatesContainPoint(coordinates: Coordinates[], coordinate: Coordinate): Boolean
```

 But looking at the [code](https://github.com/Esri/terraformer/blob/master/terraformer.js#L504) and after testing, it seems like this should just be `Coordinates`.

**Comment by jgravois** [comment]
makes sense to me. a PR would be welcome.

**Comment by jgravois** [comment]
fixed in c014eb8. thanks for the heads up!
